normalizer/app/streamlit_app.py

```
@@ -1,211 +1,186 @@
 import streamlit as st
 import pandas as pd
-import csv
 import io
 import os
-from typing import Dict, List, Any
-from dotenv import load_dotenv
+from geo_processor import *
```

Ruff (0.11.9)

`from geo_processor import *` used; unable to detect undefined names

(F403)